gems/aws-sdk-sqs/lib/aws-sdk-sqs/queue_poller.rb

```diff
@@ -444,27 +453,25 @@ def initialize
 
         # @return [Time,nil]
         attr_accessor :polling_stopped_at
-
       end
 
       # A read-only set of configuration used by the QueuePoller.
       class PollerConfig
-
         # @api private
-        CONFIG_OPTIONS = Set.new([
-          :idle_timeout,
-          :skip_delete,
-          :before_request,
-        ])
+        CONFIG_OPTIONS = Set.new(%i[
+                                   idle_timeout
```

Comment:
Ditto on spacing here - I do like the %i, but in general the SDK has used the less indented style

Reply:
I've found rubocop will preserve this spacing not just with curly brace but also the parenthesis. So I believe it will respect something like

```
CONFIG_OPTIONS = Set.new [
  thing
  thing
]
```